src/main/java/reactor/kafka/receiver/ImmutableReceiverOptions.java

```diff
@@ -746,10 +767,43 @@ public ReceiverOptions<K, V> maxDelayRebalance(Duration maxDelay) {
                 assignTopicPartitions,
                 subscribePattern,
                 schedulerSupplier,
-                consumerListener
+                consumerListener,
+                pauseAllAfterRebalance
             );
     }
 
+    @Override
+    public Boolean pauseAllAfterRebalance() {
+        return this.pauseAllAfterRebalance;
+    }
+
+    @Override
+    public ReceiverOptions<K, V> pauseAllAfterRebalance(Boolean pauseAll) {
```

Comment:
```suggestion
    public ReceiverOptions<K, V> pauseAllAfterRebalance(boolean pauseAll) {
```

Prevent user from setting it to `null`.